compass_reader/compass.py

```diff
@@ -0,0 +1,88 @@
+import threading
+import cv2
+from cv2 import aruco
+import numpy as np
+import time
+import math
+
+COMPASS_ID = 17
+BUFFER_SIZE = 5  # Number of values before giving a result
+WEBCAM_ID = 0
+DEBUG = True
+
+class CompassWatcher:
+    def __init__(self):
+        # Error
+        self.error = None
+
+        # Compass values
+        self.values = ""
```

Comment:
Why storing the values in a string instead of a list?

Reply:
It's easier to compare and print for debug 😉 (I'm lazy)

Reply:
As long as it works…

Reply:
Works perfectly! It's to permit this -> `if self.values == "S" * BUFFER_SIZE:`

Reply:
Otherwise I need a loop… it's boring ^^

Reply:
Yes, I saw this horror… You are now a python master, I guess!

Reply:
I still prefer the JS version ❤️ 
```js
"S".repeat(bufferSize)
```